hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/ReturnTypes.java

```diff
@@ -65,7 +65,7 @@ private ReturnTypes() {
             + ")[],";
     protected static final String EXPIRY_FIELDS =
             // second, autoRenewAccount, autoRenewPeriod
-            "(uint32,address,uint32)";
+            "(int64,address,int64)";
```

Comment:
This is the way it is defined in the solidity interface. I guess we haven't used the previous versions to compile for e2e tests.

Reply:
@stoyanov-st and @lukelee-sl is this effectively a version change. Won't this change the return type?

Reply:
Good point, @stoyanov-st, will not existing contracts break because of this change?

Reply:
This is curious as the expiry fields have been declared as `int64` since 2022. I haven't seen it represented in the response signature which is even more curious. I'll try some tests with the old version to see if it is a problem but I'm not sure how would we support both return types with the same function signature atm.